src/hotspot/cpu/x86/x86.ad

```diff
@@ -1496,7 +1495,7 @@ const bool Matcher::match_rule_supported(int opcode) {
     case Op_VectorLoadMask:
     case Op_VectorStoreMask:
     case Op_VectorBlend:
-      if (UseSSE < 4) {
+      if (UseSSE < 3) {
```

Comment:
This doesnt look correct. packusdw needs SSE4.  Also all the others in this case block need SSE4.

Reply:
Thanks, DONE